cache/lib/cache_web/plugs/auth_plug.ex

```diff
@@ -36,6 +35,21 @@ defmodule CacheWeb.Plugs.AuthPlug do
     end
   end
 
+  defp extract_handles(%{"project_id" => project_id}) when is_binary(project_id) do
```

Comment:
is this still relevant? At least the new module cache controller API is using `account_handle` and `project_handle` instead of `project_id`